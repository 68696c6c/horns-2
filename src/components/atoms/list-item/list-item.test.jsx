/* global describe, it, expect */
import React from 'react'

import { render } from 'setup-test'

import ListItem from '.'

describe('ListItem', () => {
  it('should render as default', () => {
    const { container } = render(
      <ul>
        <ListItem>example</ListItem>
        <ListItem>example</ListItem>
      </ul>
    )
    expect(container).toMatchSnapshot()
  })
})
