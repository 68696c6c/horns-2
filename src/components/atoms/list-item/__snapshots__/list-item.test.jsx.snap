// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItem should render as default 1`] = `
.emotion-0 {
  font-family: Helvetica,sans-serif;
  font-size: medium;
  font-style: normal;
  font-weight: 400;
  line-height: 1.6em;
  margin-top: 0px;
  margin-bottom: 0px;
}

<div>
  <ul>
    <li
      class="list-item emotion-0"
    >
      example
    </li>
    <li
      class="list-item emotion-0"
    >
      example
    </li>
  </ul>
</div>
`;
